src/App.tsx

```diff
@@ -102,24 +99,30 @@ export class App extends Component<{}, State> {
         <div className="buttons">
           <button
             type="button"
-            className="button is-info is-light"
-            onClick={this.sortAlphabetic}
+            className={sortType === SortType.ALPHABET
+              ? 'button is-info'
+              : 'button is-info is-light'}
```

Comment:
```
className={classNames(
              'button is-info',
              { 'button is-info is-light': sortType !== SortType.ALPABET },
            )}
```
It might be better to import classNames from here [https://www.npmjs.com/package/classnames](url) and use it.